pkg/kubelet/apis/config/validation/validation.go

```diff
@@ -158,5 +159,8 @@ func ValidateKubeletConfiguration(kc *kubeletconfig.KubeletConfiguration) error
 		allErrors = append(allErrors, err)
 	}
 	allErrors = append(allErrors, metrics.ValidateShowHiddenMetricsVersion(kc.ShowHiddenMetricsForVersion)...)
+	if err := logs.ValidateLogFormat(kc.LogFormat); err != nil {
+		allErrors = append(allErrors, err)
+	}
```

Comment:
```suggestion
   o := logs.NewOptions()
   o 
	if err := logs.ValidateLogFormat(kc.LogFormat); err != nil {
		allErrors = append(allErrors, err)
	}
```
I don't think `ValidateLogFormat` function will be used anywhere beside Kubelet. I don't think we should add it to logs/options.go, but just inline

Reply:
```

	logOption := logs.NewOptions()
	if kc.LogFormat != "" {
		logOption.LogFormat = kc.LogFormat
	}
	allErrors = append(allErrors, logOption.Validate()...)
```
change `o` var to `logOption` for more readability, is it ok?